### PR TITLE
reSTify PEP 285

### PR DESCRIPTION
**Pull request by mozillazg**
refer #4

### CODE REVIEW

File: pep-0285.txt [hunk 1]
```diff
@@ -5,437 +5,453 @@ Last-Modified: $Date$
...
+"doubletons"?  The real implementation will not allow other
+instances of bool to be created.
+
+``True`` and ``False`` will properly round-trip through pickling and
+marshalling; for example ``pickle.loads(pickle.dumps(True))`` will
+return ``True``, and so will ``marshal.loads(marshal.dumps(True))``.
+
+All built-in operations that are defined to return a Boolean
+result will be changed to return False or True instead of 0 or 1.
+In particular, this affects comparisons (<, <=, ==, !=, >, >=, is,
```
Comment (brettcannon) [inline pep-0285.txt]: Probably should make the comparison operators literals.

File: pep-0285.txt [hunk 2]
```diff
@@ -5,437 +5,453 @@ Last-Modified: $Date$
...
-    - Other languages (C99, C++, Java) name the constants "false" and
-      "true", in all lowercase.  For Python, I prefer to stick with
-      the example set by the existing built-in constants, which all
-      use CapitalizedWords: None, Ellipsis, NotImplemented (as well as
-      all built-in exceptions).  Python's built-in namespace uses all
-      lowercase for functions and types only.
+- Other languages (C99, C++, Java) name the constants "false" and
+  "true", in all lowercase.  For Python, I prefer to stick with
+  the example set by the existing built-in constants, which all
+  use CapitalizedWords: ``None``, Ellipsis, ``NotImplemented`` (as well as
```
Comment (brettcannon) [inline pep-0285.txt]: `Ellipsis` should be made a literal.

**Comment by mozillazg** [comment]
@brettcannon Thanks for your review! Please check again.

**Comment by gvanrossum** [comment]
Looks good now.

**Pull request merged**